codegen/Package.swift

```diff
@@ -0,0 +1,109 @@
+// swift-tools-version:5.7
+
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+ 
+import PackageDescription
```

Comment:
Delete this file, it was deleted in `main` branch.  (Perhaps it snuck back in when you merged?)

Reply:
Deleted as pointed out. And yeah, I probably made a mistake of keeping file when merging changes from main.